n brokes npm

after I switch from one version to another my npm is broken...

```
MacBook-Pro-Dmitri:~ dmitri$ npm --version

module.js:340
    throw err;
          ^
Error: Cannot find module 'npmlog'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:280:25)
    at Module.require (module.js:362:17)
    at require (module.js:378:17)
    at /usr/local/bin/npm:19:11
    at Object.<anonymous> (/usr/local/bin/npm:87:3)
    at Module._compile (module.js:449:26)
    at Object.Module._extensions..js (module.js:467:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
```

then i do like adviced here - https://github.com/visionmedia/n/issues/101

```
curl https://npmjs.org/install.sh | sh
```

npm start working.. but then i do another switch for node version

```
n 0.8.22
```

and npm now is broken again..

it's very annoying..

--- a/bin/n
+++ b/bin/n
@@ -243,7 +243,7 @@ activate() {
   local dir=$VERSIONS_DIR/$version
   check_current_version
   echo $active > $VERSIONS_DIR/.prev
-  cp -fr $dir/* $N_PREFIX
+  cp -fR $dir/* $N_PREFIX
 }
 
 #

